### ISSUE TITLE
How to run this project

### ISSUE THREAD

**Issue by ghost** [yushulx/Android-IP-Camera#3]
I am having trouble to run this project kindly explain how to execute it.

**Comment by zenith85** [comment]
me too, it seems that the developer doesn't care anymore

**Comment by baigui** [comment]
I don't think you should read this project, it's too old. it's need eclipse IDE.

**Comment by maifeeulasad** [comment]
You can find the latest version here: https://github.com/maifeeulasad/Android-IP-Camera

I've created a pull request, here: https://github.com/yushulx/Android-IP-Camera/pull/5
